taperole/hosts

```diff
@@ -0,0 +1,9 @@
+[production]
+162.243.113.144 be_app_env=production be_app_branch=master
```

Comment:
I'm not aware of how you all have your project setup, but what I usually see is dev/qa env is master and production env is a branch called production. Of course it doesn't have to be setup that way, but just what I normally see. If this is intended then :shipit: